docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -110,3 +110,29 @@ Grafana's Tempo data source latest upgrade includes support for streaming respon
 To use this feature, toggle on the "Stream response" option in either the Search or TraceQL query type, and you'll get immediate visibility of incoming traces on the results table. This smooth integration makes data exploration a breeze and speeds up decision-making.
 
 {{< video-embed src="/media/docs/grafana/data-sources/tempo-streaming.mp4" >}}
+
+<!-- Matías Wenceslao Chomicki -->
+
+### Configuration page redesign for Loki and Elasticsearch
+
+Loki and Elasticsearch data source configuration pages have been redesigned to make getting started and setting up data sources as simple and easy to understand as possible. You can now find new subsections with links to configuration pages and tooltips to assist you with configuring and customizing data sources.
+
+### Lucene query parser for Elasticsearch
+
+Adding and removing filters from Elasticsearch queries has now become smarter with the introduction of a Lucene query parser to modify queries using the "Filter for value" and "Filter out value" icons displayed in the Log Details component in Explore.
```

Comment:
This is just technical detail and should not be visible for user, so I would not add it to what's new.

Reply:
I considered this visible because instead of concatenating strings we now parse them and use this package to output the updated query, so it hipothetically works better/differently. What do you think?

Reply:
Not sure if I understand what do you mean by it working differently - the functionality - add/remove filter did not change, we just made code more reliable by using parser.

Reply:
In the sense that we now process the query, which was previously ignored. In any case, I tried to find an example where it's visible and couldn't, so I'll remove it from the doc.